reference-architecture/aws-ansible/playbooks/validation.yaml

```diff
@@ -3,36 +3,15 @@
   become: no
   vars_files:
       - vars/main.yaml
+  pre_tasks:
+      - name: set fact
+        set_fact:
+           openshift_master_cluster_public_hostname: "{{ openshift_master_cluster_public_hostname }}"
+      - name: set fact
+        set_fact:
+           openshift_master_cluster_hostname: "{{ openshift_master_cluster_hostname }}"
```

Comment:
This isn't guaranteed to work, since you are setting the same variable name that you are using.